openfisca_uk/variables/finance/benefit/benefit.py

```diff
@@ -289,6 +290,18 @@ class claims_all_entitled_benefits(Variable):
         "Whether this family would claim any benefit they are entitled to"
     )
 
+    def formula(benunit, period, parameters):
+        # Return false we have any reported values in the simulation for benefits.
+        return aggr(benunit, period, [
+            "child_tax_credit_reported",
+            "working_tax_credit_reported",
+            "universal_credit_reported",
+            "housing_benefit_reported",
+            "JSA_income_reported",
+            "income_support_reported",
+            "ESA_income_reported",
+        ]).sum() < 1
```

Comment:
Why is `.sum()` needed? Won't it sum all benunits?

Reply:
Yes, and this is the idea: if we're in a simulation with some reported receipt, use that, if not, assume all claimed.